testsuite/tests/grafana/conftest.py

```diff
@@ -0,0 +1,13 @@
+"""Check for openshift configuration"""
+
+from weakget import weakget
+import pytest
+
+from testsuite.utils import warn_and_skip
+
+
+@pytest.fixture(scope="module", autouse=True)
+def require_openshift(testconfig):
```

Comment:
Is this really the only thing that would skip this? Shouldnt there be a function checking if the grafana is even deployed and configured with the current instance?

Reply:
added one more requirement mentioned in prev comment.